web/template.go

```diff
@@ -0,0 +1,50 @@
+package web
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"text/template"
+
+	"fmt"
+
+	"github.com/feedhenry/negotiator/deploy"
+	"github.com/feedhenry/negotiator/pkg/openshift"
+)
+
+// TemplateHandler is the handler for the /services/template route
+type TemplateHandler struct {
+	tl deploy.TemplateLoader
```

Comment:
would it be worth using more verbose names here, so the later code is easier to decipher? 
i.e.
```go
    templateLoader deploy.TemplateLoad
    templateDecoder deploy.TemplateDecoder
```